task/clair-scan/0.1/clair-scan.yaml

```diff
@@ -20,6 +20,8 @@ spec:
   results:
     - name: HACBS_TEST_OUTPUT
```

Comment:
At first I drop HACBS_TEST_OUTPUT from the task since CLAIR_SCAN_RESULT is used to store the vulnerability number of clair scan result, but later I realize we need to save an ERROR result in a tekton result once there is any error, so I add HACBS_TEST_OUTPUT back.

Reply:
You can always report the HACBS_TEST_OUTPUT as well in case there are any other services which are depending on this result pattern. When reporting success, you can also indicate a `note` that `CLAIR_SCAN_RESULT` has the actual scan result summary to help users get to that point if they are looking.

Reply:
Thanks! Updated.